Offline editor of .jojo-files with audio playback

olekenneth opened the pull request:
Drop the .jojo-file from main screen. This will read the Jojo-file and take you to the editor.
You can then read the transcript and choose the audio file to play back segments from the file.
Changes will be stored in the current state of the page, to save, download the file.

# Save/export
![image](https://user-images.githubusercontent.com/544388/218480658-9662089b-61f1-4a5c-ba0e-c7c75b84b4ad.png)

# Mobile
![image](https://user-images.githubusercontent.com/544388/218338222-e615db49-5070-47ff-98c4-aa1b9bcc8308.png)

# Desktop
![image](https://user-images.githubusercontent.com/544388/218338315-194518ab-8980-4551-98d1-b39d2bda53db.png)
![image](https://user-images.githubusercontent.com/544388/218338242-f408ddae-d424-43ae-8945-d4d10a54c8d7.png)

3ddyBoi commented:
Drag and drop is broken for audio files, you do not get the settings modal.

3ddyBoi commented:
You cant choose a jojo file when you click the choose file input button. Is this something that should be possible maybe?

3ddyBoi commented:
The save button pushes the transcription box downwards.
We should use click instead of hover or perhaps the box could lie above the transcription field when you hover it?

3ddyBoi commented:
Should the sound stop automatically when you have played the section of sound?

olekenneth commented:
> Should the sound stop automatically when you have played the section of sound?

It could. But atm it will not. 

> The save button pushes the transcription box downwards.
> 
> We should use click instead of hover or perhaps the box could lie above the transcription field when you hover it?

This it's like this on mobile. Sort of intended. On desktop it's not an issue. 

> You cant choose a jojo file when you click the choose file input button. Is this something that should be possible maybe?

Fixed

> Drag and drop is broken for audio files, you do not get the settings modal.

Fixed

3ddyBoi commented on src/static/src/components/Table.js:
```diff
@@ -0,0 +1,83 @@
+    const clickListener = (event) => {
+      const el = event.target;
+      const tr = el.closest("tr");
+      const playBtn = el.closest("svg");
```
This will need some more logic to work than this, but should we do this?
```suggestion
      const playBtn = el.closest("svg");
      if(!audio){
      	alert("you need to upload a soundfile to listen)
      	}
      }
```

3ddyBoi reviewed (approved):
Looks good 💯
